client/state/selectors/get-site-migration-status.js

```diff
@@ -13,6 +18,11 @@ import getRawSite from 'state/selectors/get-raw-site';
  */
 export default function getSiteMigrationStatus( state, siteId ) {
 	const site = getRawSite( state, siteId );
+	let status = 'inactive';
+
+	if ( site ) {
```

Comment:
I don't think you need the check here. `get` should handle the `site === null` case and return the default value.

Reply:
Thanks!